src/style.css

```diff
@@ -0,0 +1,34 @@
+* {
+  margin: 0;
+}
```

Comment:
do not reset margins like this. only do it for the tags that you actually need to reset margins for

Reply:
Owkay noted.... How do I reset the browser's default margin? Do I use the html or body selector? and do I set it to 0 or 8px?

Reply:
Okay thanks had to set the body margin to 0 for the test to pass

Reply:
generally you want to reset margins for tag that you know you need margin 0.
for example: 

`body,
  p,
  ul {
    margin: 0;
  }`

something like that

Reply:
but yeah, in this case you only needed to reset it for the body